lib/models/message.ts

```diff
@@ -281,4 +293,52 @@ export class MessageModel extends DocumentDbModel<
       query: "SELECT * FROM messages m WHERE (m.fiscalCode = @fiscalCode)"
     });
   }
+
+  public async attachStoredContent(
```

Comment:
add a short comment on the purpose of the function

Reply:
ok